packages/gatsby-source-s3/package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "gatsby-source-s3",
   "description": "A Gatsby plugin to source objects and images from AWS S3",
-  "version": "3.2.4",
+  "version": "4.0.0",
```

Comment:
This will be handled by CI please revert the version change and run 'yarn cs' from the repo root to add a changelog and select major version release.

Reply:
Is there a pattern for additional commit messages? I kind of guessed based on what I saw in the project history for the first message.

Reply:
Commits, won't really, I use semantic commits but not required. For the change log please clearly state what changed and specifically what was a breaking change and point to the updated docs for config. The changeset file is markdown so you can always do more formatting and content after creating the file with the cli command.

Reply:
I've reverted the version change and added a changelog with `yarn cs`.